konrad/core.py

```diff
@@ -350,6 +375,32 @@ def run(self):
 
             # Calculate temperature change for convergence check.
             self.deltaT = (self.atmosphere['T'] - T) / self.timestep_days
+            
+            # Calculate the delta for the radiative flux at the TOA.
+            self.delta2 = (4*constants.stefan_boltzmann*
+                           (self.atmosphere['T'][0,-1]**3)*
+                           self.delta
+                          )
+            
+            # Calculate the radiative flux change at the TOA to check
+            # convergence.
+            self.deltaN = (
+                           (
+                            (self.radiation["sw_flxd"][0,-1] +
+                             self.radiation["lw_flxd"][0,-1]
+                            ) -
+                            (self.radiation["sw_flxu"][0,-1] +
+                             self.radiation["lw_flxu"][0,-1]
+                            )
+                           ) - N
+                          ) / self.timestep_days
+            if self.logevery is not None and self.niter % self.logevery == 0:
```

Comment:
Add some point I should add an `RCE.log()` utility method. But this is merely a comment to myself ;)